pkg/sql/catalog/systemschema/system.go

```diff
@@ -637,6 +637,17 @@ CREATE TABLE system.span_configurations (
     CONSTRAINT check_bounds CHECK (start_key < end_key),
     FAMILY "primary" (start_key, end_key, config)
 )`
+
+	SingleVersionDescriptorLeasesTableSchema = `
+CREATE TABLE system.single_version_descriptor_leases (
+    action        STRING NOT NULL,
+    descriptor_id INT8 NOT NULL,
+    session_id    BYTES NOT NULL,
+
+    CONSTRAINT "primary" PRIMARY KEY (action, descriptor_id, session_id),
+    CONSTRAINT check_action CHECK (action IN ('notify', 'lease')),
```

Comment:
Do we want to be that strict? I'm a big fan of avoiding complexity cause "yagni"... however, I ams
concerned that every code change or issue around leasing protocol would necessitate
non-backportable changes with required schema migrations.

Would it be better to keep action w/out contraint, possibly changing it to an int, and enforce things through the code (via enum or whatnot)?

Reply:
Yeah definitely should remove it. I already added another 🤦‍♂️